### DISCUSSION TITLE
Global menu is not shown in KDE

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [kovidgoyal/kitty#6174] [Q&A] [answered]
I am using Manjaro with KDE Plasma 5.27.3, and I find that global menus of kitty are not shown. How to enable it?

**Comment by kovidgoyal** [comment]
kitty has no support for global menus other than in macOS which has an
actual defined API for this.

**Reply by ChenZhongPu** [reply]
Without the support for global menus, how can I change the settings? Or, is there a default keymap to open the settings of Kitty in Linux?

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/conf/
